Userland/Utilities/cut.cpp

```diff
@@ -134,6 +134,19 @@ static void process_line_bytes(StringView line, Vector<Range> const& ranges)
     outln();
 }
 
+static void process_line_characters(StringView line, Vector<Range> const& ranges)
+{
+    for (auto& i : ranges) {
+        if (i.m_from >= line.length())
+            continue;
+
+        auto to = min(i.m_to, line.length());
+        auto sub_string = ByteString(line).substring(i.m_from - 1, to - i.m_from + 1);
+        out("{}", sub_string);
+    }
+    outln();
+}
```

Comment:
I would expect this to work by creating a `String` or `Utf8View` from the input `StringView`, then iterating over its code points to create the substring you display.

Reply:
I tested this on ubuntu and the behavior was the same as `-b`. I agree that it makes more sense to respect the utf8-ness. it is fixed now!